src/backend_model.cc

```diff
@@ -305,15 +305,15 @@ TritonModel::ResolveBackendConfigs(
     }
 
     // add the rest of the global configs
-    if (global_index < global_backend_config.size()) {
-      auto& current_global_setting = global_backend_config.at(global_index);
+    while (global_index < global_backend_config.size()) {
```

Comment:
use `for (;global_index < global_backend_config.size(); ++global_index)` for clarity, same for below